src/main/java/jp/co/soramitsu/iroha/java/TransactionBuilder.java

```diff
@@ -140,6 +143,23 @@ public TransactionBuilder createAccount(
     );
   }
 
+  public TransactionBuilder createAccount(
+      String accountId,
+      PublicKey publicKey
+  ) {
+    if (nonNull(this.validator)) {
+      this.validator.checkAccountId(accountId);
+    }
+
+    val t = accountId.split("@");
```

Comment:
extract delimiter to a constants from here and FormValidator

Reply:
fixed